webapp/portlet/src/main/webapp/vue-apps/space-settings/components/SpaceSettings.vue

```diff
@@ -16,6 +19,7 @@
 </template>
 
 <script>
+import {getSpaceApplications} from '../utils.js';
```

Comment:
Please avoid adding an 'import' inside Vue files and move `getSpaceApplications` into SpaceService.js where you can use it using `this.$spaceService.getSpaceApplications`

Reply:
Ok